Libs/Optimize/ParticleSystem/itkParticleSystem.txx

```diff
@@ -337,21 +331,22 @@ void ParticleSystem<VDimension>::AdvancedAllParticleSplitting(double epsilon)
           // constraints to generate a new particle position.
           PointType newpos = this->GetDomain(j)->GetPositionAfterSplit(lists[j][i], random, epsilon);
```

Comment:
@akenmorris this GetDomain(j) should be changed to GetDomain(dom_to_process+j*domains_per_shape) 

this was part of the contour domain conflicts I guess

Reply:
Good catch.